### ISSUE TITLE
Won't build with Flutter 3.22.0

### ISSUE THREAD

**Issue by Emanuel-Bjurhager** [juliansteenbakker/duration_picker#25]
When building:

```
../../.pub-cache/hosted/pub.dev/duration_picker-1.1.1/lib/duration_picker.dart:132:14: Error: The getter 'headline2' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('/opt/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'headline2'.
            .headline2!
             ^^^^^^^^^
../../.pub-cache/hosted/pub.dev/duration_picker-1.1.1/lib/duration_picker.dart:147:44: Error: The getter 'bodyText2' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('/opt/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'bodyText2'.
        style: Theme.of(context).textTheme.bodyText2,
                                           ^^^^^^^^^
../../.pub-cache/hosted/pub.dev/duration_picker-1.1.1/lib/duration_picker.dart:546:29: Error: The getter 'subtitle1' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('/opt/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'subtitle1'.
    final style = textTheme.subtitle1;
                            ^^^^^^^^^
../../.pub-cache/hosted/pub.dev/duration_picker-1.1.1/lib/duration_picker.dart:606:37: Error: The getter 'backgroundColor' isn't defined for the class 'ThemeData'.
 - 'ThemeData' is from 'package:flutter/src/material/theme_data.dart' ('/opt/flutter/packages/flutter/lib/src/material/theme_data.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'backgroundColor'.
        backgroundColor = themeData.backgroundColor;
                                    ^^^^^^^^^^^^^^^
Target kernel_snapshot failed: Exception
```

**Comment by redguitars97** [comment]
Hi @Emanuel-Bjurhager ,

I'm facing the same issue.

I modified the library locally to bypass the issue.

I don't know if someone will update the library.

PS. Consider that this issue has been duplicated on #26 .

Thanks and best regards,
Mattia.

**Comment by juliansteenbakker** [comment]
Fixed with version v1.2.0. Thanks for raising this issue.
